content/blog/2022/05/camel317-whatsnew/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Apache Camel 3.17 What's New"
+date: 2022-03-23
+authors: [davsclaus]
+categories: ["Releases"]
+preview: Details of what we have done in the Camel 3.17 release.
+---
+
+Apache Camel 3.17 has just been TODO: released(/blog/2022/05/RELEASE-3.17.0/).
+
+This release introduces a set of new features and noticeable improvements that we will cover in this blog post.
+
+### Java 17 (runtime) support
+
+This is the first release where we have official Java 17 support. The support is runtime only,
```

Comment:
Really easy to remember. Java 17 support is introduced in Camel 3.17 :)

Reply:
Ah of course, lets add a sentence about this, you are welcome to add that.